velox/functions/sparksql/Arithmetic.h

```diff
@@ -262,6 +262,13 @@ struct Log1pFunction {
   }
 };
 
+template <typename T>
```

Comment:
T is not in use. Do we need it?

Reply:
Not yet and removed. 
But just find all other functions described with this. 
Another purpose is to forward compatible with later c++ version, which can provide expm1 with float/double or other floating type.